src/components/login-view/login-view.jsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from "react";
+
+export const LoginView = ({ onLoggedIn }) => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const handleSubmit = (event) => {
+    //prevents the default behaviour of the form which is to reload the entire page
+    event.preventDefault();
+
+    const data = {
+      Username: username,
+      Password: password,
+    };
+
+    fetch("https://movie-geeks-one.vercel.app/login", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(data),
+    })
+    .then((response) => response.json())
+    .then((data) => {
+        console.log("Login response: ", data);
+        if (data.user) {
+            localStorage.setItem("user", JSON.stringify(data.user));
+            localStorage.setItem("token", data.token);
+            onLoggedIn(data.user, data.token);
+        } else {
+            alert("No such user");
+        }
+    })
+    .catch((e) => {
+        alert("Something went wrong")
+    });
+  };
+  return (
+    <form onSubmit={handleSubmit}>
+      <label>
+        Username:
+        <input
+          type="text"
+          value={username}
+          onChange={(e) => setUsername(e.target.value)}
+          required
+        />
+      </label>
+      <label>
+        Password:
+        <input
+          type="password"
+          value={password}
+          onChange={(e) => setPassword(e.target.value)}
+          minLength="5"
```

Comment:
This should not be here. You also aren’t enforcing this in the `<SignupView />` component. This causes an issue because your API doesn’t enforce these validations. I signed up with a 4 letter password and so was blocked from logging in.